client/src/common/components/Filter.jsx

```diff
@@ -1,7 +1,10 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import _debounce from 'lodash/debounce';
-import _trim from 'lodash/trim';
+import _trimStart from 'lodash/trimStart';
```

Comment:
Please trim only before performing the search and always leave the input as it is entered by the user (make sure filter "list 2   " ["list 1", "list 2"] yields ["list 2"]).